src/styles/index.ts

```diff
@@ -445,6 +445,9 @@ const styles = (theme: ThemeColors) =>
 
         textIvoryLight: {
```

Comment:
```suggestion
        textVersion: {
```

textIvoryLight would be misleading as such names should refer to styles that are about colors only

Reply:
It was adressed, but I forgot to comment it.